tests/fips/openssl/openssl_pubkey_dsa.pm

```diff
@@ -36,6 +38,10 @@ sub run {
     my $file_sig      = $file_dgt . ".sig";
     my @dsa_key_sizes = (2048, 3072);
 
+    # Add the openssl version check
+    my $openssl_version_output = script_output("openssl version");
```

Comment:
I suggest to explicit select the second field of `openssl version` output strings, to avoid other unknown number strings add to other fields in future (although which may not probably be happened)

```
my $openssl_version_output = script_output("openssl version | awk '{print $2}'");
```

Reply:
Done the modification. Thanks.